main.tf

```diff
@@ -1,7 +1,9 @@
 locals {
-  version           = "0.1.0" #TODO: Dynamically inject?
+  name = "fullstory-cloud-relay/aws"
+
+  version           = compact([for m in jsondecode(file("${path.module}/../modules.json"))["Modules"] : length(regexall(".${local.name}.*", m["Source"])) > 0 ? m["Version"] : ""])[0]
```

Comment:
Neat! Sometimes I like to run this Terraform directly from the repo (where `modules.json` won't exist), could we throw in a `try` as a fallback scenario? Something like `try(compact(...), "unreleased")`